packages/ra-data-json-server/src/index.js

```diff
@@ -82,6 +82,7 @@ export default (apiUrl, httpClient = fetchUtils.fetchJson) => {
             case GET_MANY: {
                 const query = {
                     id: params.ids,
+                    [`id_like`]: params.ids.join('|'), // Legacy
```

Comment:
I'm not sure this is a proper fix. APIs will receives a query with both filters. What are they supposed to do?

Reply:
Hadn't thought of that.
I'm not sure: should we revert the change and wait for the 3.0 to change the behavior, or adding a param and tell to users to ignore it?

Option 2: add a optional parameter to the data provider when to handle this change, like:
```js
jsonServerProvider('http://jsonplaceholder.typicode.com', { addIdParamToQuery: true })
```